### PR TITLE
WIP: add test for pathExists

### PR DESCRIPTION
**Pull request by hSloan**
Solve: https://github.com/haskell-nix/hnix/issues/437

### CODE REVIEW

File: tests/eval-compare/builtins.pathExists.nix [hunk 1]
```diff
@@ -0,0 +1 @@
+builtins.pathExists "/bin/sh/invalid-director"
```
Comment (jwiegley) [inline tests/eval-compare/builtins.pathExists.nix]: Does this path exist on all machines?

**Review by jwiegley — CHANGES_REQUESTED** [review]

**Comment by jwiegley** [comment]
BTW, GitHub now supports "draft" pull requests, so that you don't need to mark your PR with "WIP" anymore.

**Review by jwiegley — CHANGES_REQUESTED** [review]
If the goal is just a directory that doesn't exist, why about `/foo/bar/baz` or something? I'm not sure whether `/var/empty` needs to exist for this test to work or not... ?

File: tests/eval-compare/builtins.pathExists.nix (hunk 1 above)
Comment (hSloan) [reply tests/eval-compare/builtins.pathExists.nix]: No, I saw the previous test assert that a path does not exist.

**Comment by hSloan** [comment]
@jwiegley I've made use of `var/empty` here, this seems to be a convention suggested by @shlevy

**Pull request merged**